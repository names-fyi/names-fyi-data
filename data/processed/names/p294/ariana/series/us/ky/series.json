[{"y":1989,"t":8,"m":0,"f":8,"mr":null,"fr":357,"mp":0,"fp":0.0003758691975192633},{"y":1990,"t":8,"m":0,"f":8,"mr":null,"fr":354,"mp":0,"fp":0.0003766833035125718},{"y":1991,"t":8,"m":0,"f":8,"mr":null,"fr":361,"mp":0,"fp":0.0003727865796831314},{"y":1993,"t":8,"m":0,"f":8,"mr":null,"fr":364,"mp":0,"fp":0.00040092212087801946},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":520,"mp":0,"fp":0.0002501500900540324},{"y":1995,"t":10,"m":0,"f":10,"mr":null,"fr":309,"mp":0,"fp":0.0005093465084296847},{"y":1996,"t":13,"m":0,"f":13,"mr":null,"fr":257,"mp":0,"fp":0.0006619818718810469},{"y":1997,"t":10,"m":0,"f":10,"mr":null,"fr":329,"mp":0,"fp":0.0005060984867655246},{"y":1998,"t":12,"m":0,"f":12,"mr":null,"fr":302,"mp":0,"fp":0.0005885237861696911},{"y":1999,"t":15,"m":0,"f":15,"mr":null,"fr":258,"mp":0,"fp":0.0007474958887726118},{"y":2000,"t":9,"m":0,"f":9,"mr":null,"fr":397,"mp":0,"fp":0.0004398826979472141},{"y":2001,"t":20,"m":0,"f":20,"mr":null,"fr":213,"mp":0,"fp":0.001004217714400482},{"y":2002,"t":18,"m":0,"f":18,"mr":null,"fr":234,"mp":0,"fp":0.0009339490478908317},{"y":2003,"t":18,"m":0,"f":18,"mr":null,"fr":232,"mp":0,"fp":0.0009139839545039098},{"y":2004,"t":26,"m":7,"f":19,"mr":482,"fr":265,"mp":0.0003191101385849745,"fp":0.0009758101792409224},{"y":2005,"t":20,"m":0,"f":20,"mr":null,"fr":220,"mp":0,"fp":0.00101615689462453},{"y":2006,"t":21,"m":0,"f":21,"mr":null,"fr":222,"mp":0,"fp":0.0010348905972797163},{"y":2007,"t":30,"m":0,"f":30,"mr":null,"fr":156,"mp":0,"fp":0.0014654877631771774},{"y":2008,"t":26,"m":0,"f":26,"mr":null,"fr":176,"mp":0,"fp":0.0012862372613040467},{"y":2009,"t":28,"m":0,"f":28,"mr":null,"fr":165,"mp":0,"fp":0.0014530358069538142},{"y":2010,"t":26,"m":0,"f":26,"mr":null,"fr":167,"mp":0,"fp":0.0014175890082329208},{"y":2011,"t":26,"m":0,"f":26,"mr":null,"fr":175,"mp":0,"fp":0.0014166621260829292},{"y":2012,"t":28,"m":0,"f":28,"mr":null,"fr":156,"mp":0,"fp":0.00154448673396216},{"y":2013,"t":35,"m":0,"f":35,"mr":null,"fr":126,"mp":0,"fp":0.0018944519621109607},{"y":2014,"t":58,"m":0,"f":58,"mr":null,"fr":73,"mp":0,"fp":0.0031429500379321555},{"y":2015,"t":45,"m":0,"f":45,"mr":null,"fr":96,"mp":0,"fp":0.002427053556981824},{"y":2016,"t":29,"m":0,"f":29,"mr":null,"fr":148,"mp":0,"fp":0.0015923566878980893},{"y":2017,"t":26,"m":0,"f":26,"mr":null,"fr":162,"mp":0,"fp":0.0014582982780862639},{"y":2018,"t":31,"m":0,"f":31,"mr":null,"fr":139,"mp":0,"fp":0.0017526006331976482},{"y":2019,"t":35,"m":0,"f":35,"mr":null,"fr":114,"mp":0,"fp":0.002005500802200321},{"y":2020,"t":21,"m":0,"f":21,"mr":null,"fr":210,"mp":0,"fp":0.0012491077801570307},{"y":2021,"t":29,"m":0,"f":29,"mr":null,"fr":145,"mp":0,"fp":0.0017117223468303623},{"y":2022,"t":25,"m":0,"f":25,"mr":null,"fr":165,"mp":0,"fp":0.0014961996528816805}]