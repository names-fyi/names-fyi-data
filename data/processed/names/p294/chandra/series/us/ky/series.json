[{"y":1958,"t":7,"m":0,"f":7,"mr":null,"fr":445,"mp":0,"fp":0.00021271423362100402},{"y":1962,"t":7,"m":0,"f":7,"mr":null,"fr":458,"mp":0,"fp":0.00022719896137617657},{"y":1964,"t":7,"m":0,"f":7,"mr":null,"fr":453,"mp":0,"fp":0.0002348993288590604},{"y":1965,"t":8,"m":0,"f":8,"mr":null,"fr":380,"mp":0,"fp":0.0002987303958177745},{"y":1966,"t":9,"m":0,"f":9,"mr":null,"fr":342,"mp":0,"fp":0.0003624939584340261},{"y":1967,"t":5,"m":0,"f":5,"mr":null,"fr":496,"mp":0,"fp":0.0002032768223767126},{"y":1968,"t":5,"m":0,"f":5,"mr":null,"fr":489,"mp":0,"fp":0.00020870726718704347},{"y":1969,"t":10,"m":0,"f":10,"mr":null,"fr":339,"mp":0,"fp":0.00040515355319666155},{"y":1970,"t":8,"m":0,"f":8,"mr":null,"fr":404,"mp":0,"fp":0.0003078936227533387},{"y":1971,"t":10,"m":0,"f":10,"mr":null,"fr":351,"mp":0,"fp":0.0003850300323425227},{"y":1973,"t":11,"m":0,"f":11,"mr":null,"fr":320,"mp":0,"fp":0.00047421969305052594},{"y":1974,"t":8,"m":0,"f":8,"mr":null,"fr":386,"mp":0,"fp":0.0003512623490669594},{"y":1975,"t":14,"m":0,"f":14,"mr":null,"fr":283,"mp":0,"fp":0.0005983161673575794},{"y":1976,"t":15,"m":0,"f":15,"mr":null,"fr":266,"mp":0,"fp":0.0006401775425718066},{"y":1977,"t":12,"m":0,"f":12,"mr":null,"fr":320,"mp":0,"fp":0.0004880429477794046},{"y":1978,"t":17,"m":0,"f":17,"mr":null,"fr":240,"mp":0,"fp":0.000711892797319933},{"y":1979,"t":10,"m":0,"f":10,"mr":null,"fr":346,"mp":0,"fp":0.0004083299305839118},{"y":1980,"t":13,"m":0,"f":13,"mr":null,"fr":289,"mp":0,"fp":0.0005229073649491171},{"y":1981,"t":6,"m":0,"f":6,"mr":null,"fr":450,"mp":0,"fp":0.00025136154168412233},{"y":1982,"t":12,"m":0,"f":12,"mr":null,"fr":285,"mp":0,"fp":0.0005053695514845231},{"y":1983,"t":8,"m":0,"f":8,"mr":null,"fr":364,"mp":0,"fp":0.00035503483779345847},{"y":1984,"t":6,"m":0,"f":6,"mr":null,"fr":435,"mp":0,"fp":0.00027351050736199115},{"y":1985,"t":11,"m":0,"f":11,"mr":null,"fr":286,"mp":0,"fp":0.0005175739895544159},{"y":1986,"t":5,"m":0,"f":5,"mr":null,"fr":481,"mp":0,"fp":0.00023864070255822834},{"y":1987,"t":7,"m":0,"f":7,"mr":null,"fr":376,"mp":0,"fp":0.00033524904214559385},{"y":1988,"t":13,"m":0,"f":13,"mr":null,"fr":253,"mp":0,"fp":0.0006355103637074697},{"y":1989,"t":9,"m":0,"f":9,"mr":null,"fr":330,"mp":0,"fp":0.00042285284720917124},{"y":1990,"t":7,"m":0,"f":7,"mr":null,"fr":401,"mp":0,"fp":0.0003295978905735003},{"y":1991,"t":6,"m":0,"f":6,"mr":null,"fr":460,"mp":0,"fp":0.00027958993476234854},{"y":1993,"t":5,"m":0,"f":5,"mr":null,"fr":524,"mp":0,"fp":0.0002505763255487622},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":527,"mp":0,"fp":0.0002501500900540324},{"y":1995,"t":9,"m":0,"f":9,"mr":null,"fr":340,"mp":0,"fp":0.00045841185758671625},{"y":1997,"t":6,"m":0,"f":6,"mr":null,"fr":480,"mp":0,"fp":0.0003036590920593147}]