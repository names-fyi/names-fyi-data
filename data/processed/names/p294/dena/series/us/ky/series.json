[{"y":1939,"t":5,"m":0,"f":5,"mr":null,"fr":458,"mp":0,"fp":0.00018134996917050524},{"y":1953,"t":7,"m":0,"f":7,"mr":null,"fr":415,"mp":0,"fp":0.0002127659574468085},{"y":1954,"t":6,"m":0,"f":6,"mr":null,"fr":473,"mp":0,"fp":0.00017589118198874296},{"y":1955,"t":8,"m":0,"f":8,"mr":null,"fr":394,"mp":0,"fp":0.0002379182156133829},{"y":1956,"t":9,"m":0,"f":9,"mr":null,"fr":378,"mp":0,"fp":0.00026828032312874476},{"y":1957,"t":11,"m":0,"f":11,"mr":null,"fr":348,"mp":0,"fp":0.00032609984584370925},{"y":1958,"t":9,"m":0,"f":9,"mr":null,"fr":386,"mp":0,"fp":0.00027348972894129087},{"y":1959,"t":12,"m":0,"f":12,"mr":null,"fr":328,"mp":0,"fp":0.0003638127577007034},{"y":1960,"t":13,"m":0,"f":13,"mr":null,"fr":319,"mp":0,"fp":0.00040625},{"y":1961,"t":18,"m":0,"f":18,"mr":null,"fr":266,"mp":0,"fp":0.0005645287752861848},{"y":1962,"t":20,"m":0,"f":20,"mr":null,"fr":252,"mp":0,"fp":0.0006491398896462187},{"y":1963,"t":15,"m":0,"f":15,"mr":null,"fr":295,"mp":0,"fp":0.0004970178926441351},{"y":1964,"t":16,"m":0,"f":16,"mr":null,"fr":279,"mp":0,"fp":0.0005369127516778523},{"y":1965,"t":20,"m":0,"f":20,"mr":null,"fr":233,"mp":0,"fp":0.0007468259895444362},{"y":1966,"t":14,"m":0,"f":14,"mr":null,"fr":272,"mp":0,"fp":0.000563879490897374},{"y":1967,"t":18,"m":0,"f":18,"mr":null,"fr":236,"mp":0,"fp":0.0007317965605561653},{"y":1968,"t":7,"m":0,"f":7,"mr":null,"fr":405,"mp":0,"fp":0.00029219017406186085},{"y":1969,"t":29,"m":0,"f":29,"mr":null,"fr":172,"mp":0,"fp":0.0011749453042703185},{"y":1970,"t":23,"m":0,"f":23,"mr":null,"fr":213,"mp":0,"fp":0.0008851941654158488},{"y":1971,"t":32,"m":0,"f":32,"mr":null,"fr":167,"mp":0,"fp":0.0012320961034960726},{"y":1972,"t":25,"m":0,"f":25,"mr":null,"fr":195,"mp":0,"fp":0.0010524543234823609},{"y":1973,"t":17,"m":0,"f":17,"mr":null,"fr":248,"mp":0,"fp":0.0007328849801689946},{"y":1974,"t":15,"m":0,"f":15,"mr":null,"fr":264,"mp":0,"fp":0.0006586169045005489},{"y":1975,"t":13,"m":0,"f":13,"mr":null,"fr":297,"mp":0,"fp":0.0005555792982606094},{"y":1976,"t":16,"m":0,"f":16,"mr":null,"fr":255,"mp":0,"fp":0.000682856045409927},{"y":1977,"t":12,"m":0,"f":12,"mr":null,"fr":321,"mp":0,"fp":0.0004880429477794046},{"y":1978,"t":12,"m":0,"f":12,"mr":null,"fr":302,"mp":0,"fp":0.0005025125628140704},{"y":1979,"t":11,"m":0,"f":11,"mr":null,"fr":322,"mp":0,"fp":0.00044916292364230296},{"y":1980,"t":8,"m":0,"f":8,"mr":null,"fr":390,"mp":0,"fp":0.00032178914766099514},{"y":1981,"t":8,"m":0,"f":8,"mr":null,"fr":390,"mp":0,"fp":0.00033514872224549644},{"y":1984,"t":5,"m":0,"f":5,"mr":null,"fr":497,"mp":0,"fp":0.0002279254228016593},{"y":1985,"t":5,"m":0,"f":5,"mr":null,"fr":480,"mp":0,"fp":0.0002352609043429163},{"y":1986,"t":5,"m":0,"f":5,"mr":null,"fr":489,"mp":0,"fp":0.00023864070255822834}]