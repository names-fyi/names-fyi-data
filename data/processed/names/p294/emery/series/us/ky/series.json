[{"y":1912,"t":9,"m":9,"f":0,"mr":156,"fr":null,"mp":0.0010207553589656345,"fp":0},{"y":1913,"t":8,"m":8,"f":0,"mr":203,"fr":null,"mp":0.0007527286413248024,"fp":0},{"y":1914,"t":11,"m":11,"f":0,"mr":194,"fr":null,"mp":0.0008270054883091496,"fp":0},{"y":1915,"t":14,"m":14,"f":0,"mr":203,"fr":null,"mp":0.000746626846568183,"fp":0},{"y":1916,"t":17,"m":17,"f":0,"mr":181,"fr":null,"mp":0.0008625501040133949,"fp":0},{"y":1917,"t":10,"m":10,"f":0,"mr":248,"fr":null,"mp":0.0005026388539834129,"fp":0},{"y":1918,"t":10,"m":10,"f":0,"mr":289,"fr":null,"mp":0.0004336701504835422,"fp":0},{"y":1919,"t":12,"m":12,"f":0,"mr":260,"fr":null,"mp":0.0005264312349199386,"fp":0},{"y":1920,"t":10,"m":10,"f":0,"mr":297,"fr":null,"mp":0.0004014774369680424,"fp":0},{"y":1921,"t":17,"m":17,"f":0,"mr":217,"fr":null,"mp":0.0006458721173207705,"fp":0},{"y":1922,"t":11,"m":11,"f":0,"mr":280,"fr":null,"mp":0.0004243336033638082,"fp":0},{"y":1923,"t":14,"m":14,"f":0,"mr":236,"fr":null,"mp":0.0005420264044291301,"fp":0},{"y":1924,"t":12,"m":12,"f":0,"mr":274,"fr":null,"mp":0.00044540123227674265,"fp":0},{"y":1925,"t":12,"m":12,"f":0,"mr":269,"fr":null,"mp":0.0004631773969430292,"fp":0},{"y":1926,"t":14,"m":14,"f":0,"mr":238,"fr":null,"mp":0.0005480095510236036,"fp":0},{"y":1927,"t":8,"m":8,"f":0,"mr":352,"fr":null,"mp":0.0003006501559622684,"fp":0},{"y":1928,"t":13,"m":13,"f":0,"mr":244,"fr":null,"mp":0.0005060137791444475,"fp":0},{"y":1929,"t":11,"m":11,"f":0,"mr":268,"fr":null,"mp":0.0004483939344529594,"fp":0},{"y":1930,"t":14,"m":14,"f":0,"mr":226,"fr":null,"mp":0.000545979252788394,"fp":0},{"y":1931,"t":12,"m":12,"f":0,"mr":259,"fr":null,"mp":0.0004850248575239481,"fp":0},{"y":1932,"t":12,"m":12,"f":0,"mr":255,"fr":null,"mp":0.0004576309968728549,"fp":0},{"y":1934,"t":15,"m":15,"f":0,"mr":228,"fr":null,"mp":0.0005659096053723685,"fp":0},{"y":1935,"t":10,"m":10,"f":0,"mr":289,"fr":null,"mp":0.00038230683946935813,"fp":0},{"y":1936,"t":8,"m":8,"f":0,"mr":332,"fr":null,"mp":0.00031618053908781914,"fp":0},{"y":1937,"t":11,"m":11,"f":0,"mr":261,"fr":null,"mp":0.0004279156617132187,"fp":0},{"y":1938,"t":13,"m":13,"f":0,"mr":254,"fr":null,"mp":0.00046490004649000463,"fp":0},{"y":1939,"t":6,"m":6,"f":0,"mr":398,"fr":null,"mp":0.0002196032501281019,"fp":0},{"y":1940,"t":9,"m":9,"f":0,"mr":314,"fr":null,"mp":0.00031521434575511347,"fp":0},{"y":1941,"t":7,"m":7,"f":0,"mr":351,"fr":null,"mp":0.00024599381501265113,"fp":0},{"y":1942,"t":7,"m":7,"f":0,"mr":368,"fr":null,"mp":0.00023347341738376358,"fp":0},{"y":1943,"t":9,"m":9,"f":0,"mr":309,"fr":null,"mp":0.0003040437823046519,"fp":0},{"y":1945,"t":7,"m":7,"f":0,"mr":331,"fr":null,"mp":0.00025395443331882166,"fp":0},{"y":1947,"t":5,"m":5,"f":0,"mr":471,"fr":null,"mp":0.0001330565756559689,"fp":0},{"y":1949,"t":9,"m":9,"f":0,"mr":322,"fr":null,"mp":0.0002498681251561676,"fp":0},{"y":1950,"t":6,"m":6,"f":0,"mr":400,"fr":null,"mp":0.0001679684219366759,"fp":0},{"y":1951,"t":10,"m":10,"f":0,"mr":305,"fr":null,"mp":0.00027219031546857564,"fp":0},{"y":1954,"t":5,"m":5,"f":0,"mr":414,"fr":null,"mp":0.0001355160450997398,"fp":0},{"y":1960,"t":6,"m":6,"f":0,"mr":394,"fr":null,"mp":0.00016919519485646608,"fp":0},{"y":1972,"t":5,"m":5,"f":0,"mr":388,"fr":null,"mp":0.00018840907378099328,"fp":0},{"y":2004,"t":10,"m":5,"f":5,"mr":631,"fr":729,"mp":0.00022793581327498177,"fp":0.00025679215243182166},{"y":2006,"t":12,"m":5,"f":7,"mr":579,"fr":550,"mp":0.0002085679723021733,"fp":0.00034496353242657207},{"y":2007,"t":6,"m":0,"f":6,"mr":null,"fr":606,"mp":0,"fp":0.0002930975526354355},{"y":2008,"t":17,"m":0,"f":17,"mr":null,"fr":263,"mp":0,"fp":0.0008410012862372613},{"y":2009,"t":20,"m":5,"f":15,"mr":631,"fr":297,"mp":0.00021361131285512882,"fp":0.0007784120394395433},{"y":2010,"t":22,"m":7,"f":15,"mr":483,"fr":285,"mp":0.0003116096866096866,"fp":0.0008178398124420697},{"y":2011,"t":28,"m":0,"f":28,"mr":null,"fr":162,"mp":0,"fp":0.0015256361357816161},{"y":2012,"t":31,"m":0,"f":31,"mr":null,"fr":138,"mp":0,"fp":0.0017099674554581059},{"y":2013,"t":21,"m":0,"f":21,"mr":null,"fr":207,"mp":0,"fp":0.0011366711772665765},{"y":2014,"t":20,"m":0,"f":20,"mr":null,"fr":235,"mp":0,"fp":0.0010837758751490192},{"y":2015,"t":39,"m":0,"f":39,"mr":null,"fr":112,"mp":0,"fp":0.0021034464160509143},{"y":2016,"t":34,"m":0,"f":34,"mr":null,"fr":125,"mp":0,"fp":0.0018669009444322424},{"y":2017,"t":46,"m":0,"f":46,"mr":null,"fr":86,"mp":0,"fp":0.002580066184306467},{"y":2018,"t":64,"m":5,"f":59,"mr":671,"fr":60,"mp":0.00023953243269138642,"fp":0.0033355947535052013},{"y":2019,"t":60,"m":6,"f":54,"mr":597,"fr":70,"mp":0.00029380080305552837,"fp":0.003094201237680495},{"y":2020,"t":58,"m":0,"f":58,"mr":null,"fr":56,"mp":0,"fp":0.0034499167261479893},{"y":2021,"t":54,"m":0,"f":54,"mr":null,"fr":64,"mp":0,"fp":0.0031873450596151577},{"y":2022,"t":57,"m":0,"f":57,"mr":null,"fr":58,"mp":0,"fp":0.0034113352085702315}]