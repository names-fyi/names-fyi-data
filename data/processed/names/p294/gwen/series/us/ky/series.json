[{"y":1935,"t":7,"m":0,"f":7,"mr":null,"fr":398,"mp":0,"fp":0.00026317768253252124},{"y":1942,"t":8,"m":0,"f":8,"mr":null,"fr":377,"mp":0,"fp":0.00027281407720638383},{"y":1943,"t":5,"m":0,"f":5,"mr":null,"fr":489,"mp":0,"fp":0.00017263405034008907},{"y":1944,"t":6,"m":0,"f":6,"mr":null,"fr":437,"mp":0,"fp":0.00021613832853025936},{"y":1945,"t":6,"m":0,"f":6,"mr":null,"fr":423,"mp":0,"fp":0.00022333891680625348},{"y":1946,"t":8,"m":0,"f":8,"mr":null,"fr":373,"mp":0,"fp":0.00024663953631767175},{"y":1948,"t":5,"m":0,"f":5,"mr":null,"fr":511,"mp":0,"fp":0.00014399262757746803},{"y":1949,"t":7,"m":0,"f":7,"mr":null,"fr":418,"mp":0,"fp":0.00020547744151231396},{"y":1950,"t":5,"m":0,"f":5,"mr":null,"fr":510,"mp":0,"fp":0.00014904018123286037},{"y":1952,"t":10,"m":0,"f":10,"mr":null,"fr":343,"mp":0,"fp":0.0002968592293534406},{"y":1954,"t":5,"m":0,"f":5,"mr":null,"fr":529,"mp":0,"fp":0.00014657598499061915},{"y":1956,"t":12,"m":0,"f":12,"mr":null,"fr":320,"mp":0,"fp":0.000357707097504993},{"y":1957,"t":11,"m":0,"f":11,"mr":null,"fr":350,"mp":0,"fp":0.00032609984584370925},{"y":1958,"t":12,"m":0,"f":12,"mr":null,"fr":328,"mp":0,"fp":0.00036465297192172114},{"y":1959,"t":13,"m":0,"f":13,"mr":null,"fr":312,"mp":0,"fp":0.00039413048750909534},{"y":1960,"t":17,"m":0,"f":17,"mr":null,"fr":285,"mp":0,"fp":0.00053125},{"y":1961,"t":12,"m":0,"f":12,"mr":null,"fr":343,"mp":0,"fp":0.0003763525168574565},{"y":1962,"t":14,"m":0,"f":14,"mr":null,"fr":303,"mp":0,"fp":0.00045439792275235314},{"y":1963,"t":6,"m":0,"f":6,"mr":null,"fr":499,"mp":0,"fp":0.00019880715705765408},{"y":1964,"t":10,"m":0,"f":10,"mr":null,"fr":364,"mp":0,"fp":0.0003355704697986577},{"y":1965,"t":7,"m":0,"f":7,"mr":null,"fr":422,"mp":0,"fp":0.00026138909634055267},{"y":1966,"t":10,"m":0,"f":10,"mr":null,"fr":322,"mp":0,"fp":0.00040277106492669565},{"y":1967,"t":5,"m":0,"f":5,"mr":null,"fr":513,"mp":0,"fp":0.0002032768223767126},{"y":1968,"t":6,"m":0,"f":6,"mr":null,"fr":449,"mp":0,"fp":0.00025044872062445216},{"y":1969,"t":8,"m":0,"f":8,"mr":null,"fr":407,"mp":0,"fp":0.00032412284255732923},{"y":1971,"t":7,"m":0,"f":7,"mr":null,"fr":445,"mp":0,"fp":0.0002695210226397659},{"y":1972,"t":6,"m":0,"f":6,"mr":null,"fr":469,"mp":0,"fp":0.00025258903763576663},{"y":2022,"t":8,"m":0,"f":8,"mr":null,"fr":535,"mp":0,"fp":0.00047878388892213776}]