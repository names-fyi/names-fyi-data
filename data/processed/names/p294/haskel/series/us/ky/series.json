[{"y":1916,"t":8,"m":8,"f":0,"mr":303,"fr":null,"mp":0.00040590593130042114,"fp":0},{"y":1922,"t":8,"m":8,"f":0,"mr":368,"fr":null,"mp":0.0003086062569918605,"fp":0},{"y":1924,"t":8,"m":8,"f":0,"mr":359,"fr":null,"mp":0.00029693415485116173,"fp":0},{"y":1926,"t":12,"m":12,"f":0,"mr":275,"fr":null,"mp":0.0004697224723059459,"fp":0},{"y":1927,"t":5,"m":5,"f":0,"mr":516,"fr":null,"mp":0.00018790634747641776,"fp":0},{"y":1928,"t":7,"m":7,"f":0,"mr":378,"fr":null,"mp":0.0002724689580008563,"fp":0},{"y":1929,"t":10,"m":10,"f":0,"mr":282,"fr":null,"mp":0.0004076308495026904,"fp":0},{"y":1930,"t":8,"m":8,"f":0,"mr":339,"fr":null,"mp":0.0003119881444505109,"fp":0},{"y":1931,"t":5,"m":5,"f":0,"mr":447,"fr":null,"mp":0.00020209369063497838,"fp":0},{"y":1936,"t":6,"m":6,"f":0,"mr":408,"fr":null,"mp":0.00023713540431586437,"fp":0},{"y":1937,"t":6,"m":6,"f":0,"mr":389,"fr":null,"mp":0.00023340854275266475,"fp":0},{"y":1938,"t":6,"m":6,"f":0,"mr":409,"fr":null,"mp":0.000214569252226156,"fp":0},{"y":1940,"t":6,"m":6,"f":0,"mr":421,"fr":null,"mp":0.00021014289717007564,"fp":0},{"y":1945,"t":5,"m":5,"f":0,"mr":425,"fr":null,"mp":0.00018139602379915832,"fp":0}]