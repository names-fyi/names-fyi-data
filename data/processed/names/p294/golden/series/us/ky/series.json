[{"y":1912,"t":8,"m":0,"f":8,"mr":null,"fr":269,"mp":0,"fp":0.0005869836378310955},{"y":1913,"t":5,"m":0,"f":5,"mr":null,"fr":377,"mp":0,"fp":0.0003195092338168573},{"y":1915,"t":6,"m":0,"f":6,"mr":null,"fr":429,"mp":0,"fp":0.0002543774112858778},{"y":1916,"t":7,"m":0,"f":7,"mr":null,"fr":394,"mp":0,"fp":0.00029106029106029104},{"y":1917,"t":6,"m":0,"f":6,"mr":null,"fr":431,"mp":0,"fp":0.00024233612019871562},{"y":1918,"t":8,"m":0,"f":8,"mr":null,"fr":391,"mp":0,"fp":0.0002959893443836022},{"y":1919,"t":7,"m":0,"f":7,"mr":null,"fr":410,"mp":0,"fp":0.0002589715131335553},{"y":1920,"t":7,"m":0,"f":7,"mr":null,"fr":426,"mp":0,"fp":0.00024202192027106455},{"y":1921,"t":10,"m":0,"f":10,"mr":null,"fr":355,"mp":0,"fp":0.0003167564143173899},{"y":1922,"t":7,"m":0,"f":7,"mr":null,"fr":443,"mp":0,"fp":0.00023303039382136556},{"y":1923,"t":14,"m":9,"f":5,"mr":329,"fr":549,"mp":0.00034844554570444074,"fp":0.00016601919181857423},{"y":1924,"t":6,"m":6,"f":0,"mr":447,"fr":null,"mp":0.00022270061613837133,"fp":0},{"y":1925,"t":5,"m":0,"f":5,"mr":null,"fr":532,"mp":0,"fp":0.000166761164659974},{"y":1926,"t":13,"m":6,"f":7,"mr":413,"fr":409,"mp":0.00023486123615297296,"fp":0.0002455623377534554},{"y":1928,"t":10,"m":0,"f":10,"mr":null,"fr":338,"mp":0,"fp":0.00036023054755043225},{"y":1931,"t":5,"m":0,"f":5,"mr":null,"fr":492,"mp":0,"fp":0.00019236688211757464},{"y":1932,"t":6,"m":0,"f":6,"mr":null,"fr":433,"mp":0,"fp":0.00021957110444265535},{"y":1936,"t":5,"m":0,"f":5,"mr":null,"fr":454,"mp":0,"fp":0.0001943407960199005},{"y":1938,"t":6,"m":6,"f":0,"mr":408,"fr":null,"mp":0.000214569252226156,"fp":0}]